tensorflow/compiler/tf2tensorrt/convert/utils.h

```diff
@@ -106,6 +106,11 @@ string GetLinkedTensorRTVersion();
 // TensorRT library version information {Maj, Min, Patch}.
 string GetLoadedTensorRTVersion();
 
+// Returns the number of inputs for the engine, which also correspends to the
+// number of input tensors for the network. This can differ from the number of
+// input bindings, because each profile has a set of bindings.
```

Comment:
Shall we replace the second sentence here with this:
The number of total input bindings equals the number of profile times the number of engine inputs.